lte/gateway/c/session_manager/SessionStateEnforcer.cpp

```diff
@@ -540,10 +698,15 @@ bool SessionStateEnforcer::static_rule_init() {
   GlobalRuleList.insert_rule(2, reqpdr2);
 
   // subscriber Id 1 to PDR 1 and FAR 1
-  pdr_map_.insert(std::pair<std::string, uint32_t>("imsi00000000001", 1));
+  pdr_map_.insert(std::pair<std::string, uint32_t>("000000000020895", 2));

```

Comment:
why do we have all these hardwired rules for specific IMSIs? Instead create default static rule for all IMSIs and use them. Same thing with the UE IP addresses, etc.

Reply:
I ll change this, as part of another cleanup before the demo

Reply:
For now I changed  specific rules adding, default PDR1, default PDR2 adding for all IMSI's with restricted IMSI id's whatever we given.   If it needed to be generic for every IMSI I have to change the code, where in  for every new IMSI I shud add  ",DEFAULT_UP_LINK_PDR_ID" ",DEFAULT_DOWNLINK_LINK_PDR_ID shud be added as part of the code flow during creation  ? Let me know the flow.  Please check latest "SessionStateEnforcer.cpp" file for the latest changes

Reply:
Removed subscriber check out of SMF code,  by default it adds uplink and down link PDR for every imsi